intelmq/bots/outputs/udp/output.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import socket
+import sys
+
+import intelmq.lib.utils as utils
+from intelmq.lib.bot import Bot
+
+
+class UDPBot(Bot):
+
+    def init(self):
+        self.delimiter = self.parameters.field_delimiter
+        self.header = self.parameters.header
+        self.udp_host = self.parameters.ip
+        self.udp_port = self.parameters.port
+        self.upd_address = (self.udp_host, self.udp_port)
+        self.udp = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self.raw_field = self.parameters.raw_field
+        self.format = self.parameters.format
+
+    def process(self):
+        event = self.receive_message()
+
+        if event is None:
+            self.acknowledge_message()
+            return
+
+        if self.raw_field.upper() == 'DROP':
+            del event['raw']
+
+        if self.format.upper() == 'JSON':
+            self.send(self.header + ' ' + event.to_json())
+        else:
+            self.send(self.delimited(event))
+
+    def delimited(self, event):
+        log_line = self.header
+        for key, value in event.items():
+            log_line += self.delimiter + key + ':' + str(value)
```

Comment:
When and how can that happen? Do you have an example to work on?
Should we keep the chars or simply strip them out? How relevant can the chars be to a log/syslog/alert/etc?